### PR TITLE
Add Vendor REDZ

### PR DESCRIPTION
**Pull request by ali-ugur**
Add Vendor REDZ and product TLM series

...

#### Changes

- ...
- ...

#### Checklist for Reviewers

- [ ] Title and description should be descriptive (Not just a serial number for example).
- [ ] `vendorProfileID` should not be `vendorID`.
- [ ] All devices should be listed in the vendor's `index.yaml` file.
- [ ] At least 1 image per device and should be transparent.

#### Notes for Reviewers

...

#### Release Notes

- ...

### CODE REVIEW

File: vendor/index.yaml [hunk 1]
```diff
@@ -2052,3 +2052,11 @@ vendors:
     description: Lansitec is a technology-driven company providing cutting-edge active people and asset tracking products and services. i.e., trackers, gateways and bridges, beacons and tags, and related accessories like sensors.
     logo: lansitec.png
     website: https://www.lansitec.com/
+
+  - id: redz
+    name: REDZ Smart Communication
+    description: REDZ Smart Communication Technologies is a provider for industrial coomunication devices such as LoraWAN EndNode Modems, Electricity Meter Protocol to Modbus Protocol Gateways, LoRa RF Modems, Wireless Mbus Modems, Remote IO Devices, Serial to ETH Gateways, Industrial Unmanaged Switches with or without Broadband Powerline (BPL) connection and Energy Meter Reading Optical Probes.
+    logo: redz.jpg
```
Comment (LDannijs) [inline vendor/index.yaml:2059]: @ali-ugur It seems you forgot to run our validation: https://github.com/TheThingsNetwork/lorawan-devices?tab=readme-ov-file#validation

Im getting an error that the logo should be either png or svg

File: vendor/redz/tlm-lw.yaml [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+      AS923:
+        id: tlm-lw-profile-923
+        lorawanCertified: true
+        codec: tlm-lw-codec
+      IN865-867:
+        id: tlm-lw-profile-865
+        lorawanCertified: true
+        codec: tlm-lw-codec
+
+  -
```
Comment (LDannijs) [inline vendor/redz/tlm-lw.yaml:54]: remove this hyphen as well

File: vendor/redz/tlm-lw.yaml [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+
+# Product and data sheet URLs (optional)
+productURL: https://redz-sc.com/en/product/tlm-series-lora-and-lorawan-modems/tlm354-868mhz-lorawan-endnode-modem-with-modbus-tcprtu-scheduler-with-2-x-10100base-tx-ports-1-x-rs232-and-1-x-rs485-serial-ports
+dataSheetURL: https://admin.redz-sc.com/uploads/brochure/tlm354-datasheet-en.pdf
+
+# Commercial information
+resellerURLs:
+  - name: 'Z Telemetri'
+    region:
+      - Worldwide
```
Comment (LDannijs) [inline vendor/redz/tlm-lw.yaml:123]: It errors at `Worldwide` as well. Please use something like:
```
- European Union
- United States
- Canada
- Etc
```

File: vendor/redz/tlm-lw.yaml [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+    url: https://pomiary.pl/
+msrp:
+  USD: 280
+
+# Photos
+photos:
+  main: TLM-LW.png # Image needs to have a transparent background
+  other:
+    - TLM-LW-sc1.jpg # Image needs to have a transparent background
+    - TLM-LW-sc2.jpg # Image needs to have a transparent background
```
Comment (LDannijs) [inline vendor/redz/tlm-lw.yaml:141]: You cannot use capital letters, change the name of the images to lowercase and change this.

**Comment by ali-ugur** [comment]
We have updated fixes

can you please check again

Many thanks

Regards
Ali

**Comment by LDannijs** [comment]
@ali-ugur the validation is still giving errors, please run it with 'make validate fmt' as seen here: https://github.com/TheThingsNetwork/lorawan-devices?tab=readme-ov-file#validation
It will instruct you what is wrong.

**Comment by ali-ugur** [comment]
We have updated fixes

can you please check again

Many thanks

Regards
Ali

**Review by Jaime-Trinidad — APPROVED** [review]

**Review by Jaime-Trinidad — APPROVED** [review]

**Pull request merged**